### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by sondth-1037**
## Related Tickets:
- N/A

## WHAT this PR does?
-Add CI to Project

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by sondth-1037** [comment]
ready

File: .sun-ci.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  script:
+  - yarn
+
+- name: test:phpunit
+  stage: test
+  image: sunasteriskrnd/php-workspace:7.4
+  before_script:
+  - composer install
+  script:
+  - php vendor/bin/phpunit
```
Comment (huyenttm-0901) [inline .sun-ci.yml]: end of file

**Comment by sondth-1037** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
fail CI

**Comment by sondth-1037** [comment]
ready

**Pull request merged**